cosmos/stride.json

```diff
@@ -80,6 +80,20 @@
       "coinGeckoId": "stride-staked-injective",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stride/stinj.png"
     },
+    {
+      "coinDenom": "stDYDX",
+      "coinMinimalDenom": "stadydx",
+      "coinDecimals": 18,
+      "coinGeckoId": "stride-staked-dydx",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stride/stdydx.png"
+    },
+    {
+      "coinDenom": "stTIA",
+      "coinMinimalDenom": "stutia",
+      "coinDecimals": 6,
+      "coinGeckoId": "stride-staked-tia",
```

Comment:
@riley-stride Could you remove coinGeckoId field?

Reply:
done